code/datums/diseases/_disease.dm

```diff
@@ -140,19 +140,20 @@ GLOBAL_LIST_INIT(diseases, subtypesof(/datum/disease))
 	if(spread_flags & AIRBORNE)
 		spread_range++
 
-	var/turf/T = affected_mob.loc
-	if(istype(T))
+	var/turf/target = affected_mob.loc
+	if(istype(target))
```

Comment:
Can probably swap this to an early return, and the if(curent) into an early continue, while you're here

Reply:
If the PR wasn't already 3000+ lines, that'd be good. As-is, I think it's better to avoid making even more changes if we can avoid it.